tools/ClassGen/NodeGen.cpp

```diff
@@ -460,6 +460,14 @@ int main(int argc, char **argv) {
       .setDocstring(
           "Applies ReLU, max(0, x), to each element in the Input tensor.");
 
+  BB.newNode("PRelu")
+      .addInput("Input")
+      .addInput("Slope")
+      .addResultFromCtorArg()
+      .addGradient()
```

Comment:
how do we handle PReLU gradient node? (i'd exclude it for now until support is not added).

Reply:
following our talk - removing PReluGrad node.